flink-connector-gcp-pubsub/src/main/java/org/apache/flink/connector/gcp/pubsub/sink/config/GcpPublisherConfig.java

```diff
@@ -52,7 +52,11 @@ public CredentialsProvider getCredentialsProvider() {
     }
 
     public TransportChannelProvider getTransportChannelProvider() {
-        return transportChannelProvider.getTransportChannelProvider();
+        if (transportChannelProvider==null){
+            return null;
+        } else{
```

Comment:
```suggestion
        if (transportChannelProvider == null){
            return null;
        } else {
```
is it able to compile locally? (`./mvnw clean verify`)
I guess it should fail with spotless complains